coralproject/talk
# [CORL-2251] Fix sign in bug

## What does this PR do?
This PR addresses https://vmproduct.atlassian.net/browse/CORL-2251 by displaying validation errors only when a field has actually been "dirtied" or if the form as been unsuccessfully submitted (previously they would display as soon as a field lost focus)

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
Navigate to the stream while not logged in. Click the "Sign in" button, and without typing anything into the email field, click the top half of any of Facebook/Google sign in buttons, and observe that the modal begins the OAuth flow, rather than sliding out from under your pointer.
 
## How do we deploy this PR?
No special considerations should have to be taken.
